gulpfile.js

```diff
@@ -432,9 +432,7 @@
 			logToTerminal.apply(this, arguments);
```

Comment:
@erik-ciq Can you make the same change as envOrArg here? It's only logging [object, object]. Otherwise these changes are great, thanks!

Reply:
@meredith-ciq went ahead and updated that.